modules/nf-core/dysgu/tests/main.nf.test

```diff
@@ -0,0 +1,96 @@
+nextflow_process {
+
+    name "Test Process DYSGU"
+    script "../main.nf"
+    process "DYSGU"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "dysgu"
+
+
+    test("human - bam") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test'], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.recalibrated.sorted.bam', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.recalibrated.sorted.bam.bai', checkIfExists: true)
+                ]
+                input[1] = [ [ id:'reference'], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match() },
+                { assert path(process.out.vcf.get(0).get(1)).linesGzip.contains("##fileformat=VCFv4.2") }
```

Comment:
Are the variants also unstable? You could test them using the [`nft-vcf`](https://github.com/seppinho/nft-vcf) plugin which is already enabled in this repository

Reply:
Hi @nvnieuwk,
Thank you for your comments. I apologize, but I didn't fully understand what you meant by "unstable variants."

Reply:
Sorry I wasn't clear I meant the variant lines in the VCF. Are they exactly the same if you rerun the tool or do they differ too? (Unstable files are usually caused by timestamps in the header)

Reply:
The output generated from the nf-core/sarek pipeline is identical to the output produced by running the Dysgu pipeline independently. For reference, I have attached some example screenshots below to illustrate the similarities.
PS: I still need to do some changes in the modules of `dysgu` that I haven't made yet but in theory it is working fine for now.



![Screenshot from 2024-09-26 10-06-30](https://github.com/user-attachments/assets/285dc49e-4567-44ae-9abc-95de69f95970)

![Screenshot from 2024-09-26 10-08-20](https://github.com/user-attachments/assets/24264ba6-e7a1-458b-bedd-b56e35d657c6)
  

>

Reply:
Aha that command INFO line is probably the reason for the unstable md5sum. Can you test the variants MD5 instead? See here on how to do this: https://github.com/seppinho/nft-vcf?tab=readme-ov-file#variantsmd5

Reply:
seems like I need to use

> Aha that command INFO line is probably the reason for the unstable md5sum. Can you test the variants MD5 instead? See here on how to do this: https://github.com/seppinho/nft-vcf?tab=readme-ov-file#variantsmd5

hi @nvnieuwk sorry for silly question but by adding below command line in the nf-test.config will this work, please let me know.

```
config {
  plugins {
    load "nft-vcf@1.0.7"
  }
}
```
something like this?? 
```
config {
    testsDir "."
    workDir ".nf-test"
    configFile "conf/test.config"
    profile "test"
    plugins {
      load "nft-vcf@1.0.7"
  }
}
```

Reply:
`nft-vcf` is already enabled in this repository so you shouldn't have to do this